uptane-standard.md

```diff
@@ -867,9 +867,17 @@ In order to perform full verification, an ECU SHALL perform the following steps:
 11. For each image listed in the Targets metadata file from the Director repository, locate a Targets metadata file that contains an image with exactly the same file name. For each delegated Targets metadata file that is found to contain metadata for the image currently being processed, perform all of the checks in step 10. Use the following process to locate image metadata:
     1. If the top-level Targets metadata file contains signed metadata about the image, return the metadata to be checked and skip to step 11.3.
     2. Recursively search the list of delegations, in order of appearance:
-        1. If it is a multi-role delegation {{TAP-3}}, recursively visit each role, and check that each has signed exactly the same non-custom metadata (i.e., length and hashes) about the image. If it is all the same, return the metadata to be checked and skip to step 11.3.
-        2. If it is a terminating delegation and it contains signed metadata about the image, return the metadata to be checked and skip to step 11.3. If metadata about an image is not found in a terminating delegation, return an error code indicating that the image is missing.
-        3. Otherwise, continue processing the next delegation, if any. As soon as a delegation is found that contains signed metadata about the image, return the metadata to be checked and skip to step 11.3.
+        1. If it is a multi-role delegation {{TAP-3}}, then:
+            1. First, recurse (11.2) for each role listed as if each is a single-role delegation.
+            2. Inspect the return values from each:
+                1. If a minimum number of roles in agreement for this multi-role delegation return the same non-custom metadata (i.e. length and hashes) for the image, then return that metadata for the image.  If the values returned are such that there are multiple sets of non-custom metadata meeting threshold (e.g. if the threshold is 2/4 and two sets of two roles agree on different metadata), then break the tie in favor of the earliest role in the multi-role delegation.
+                2. If the minimum number of roles in agreement threshold is not met:
+                    1. If the multi-role delegation was terminating, raise an error indicating that image info could not be verified.
+                    2. If the multi-role delegation was not terminating, return no image info and allow the process to proceed to the next delegation.
+        2. If the role specified by the delegation contains signed metadata about the image, return the metadata to be checked and continue to step 11.3.
```

Comment:
The wording here is a bit confusing. Is continuing to step 11.3 just the result of the return?

Reply:
Yeah, it's the presumptive result of the return.  I was trying to soften "skip to 11.3", but maybe we should just exclude it entirely.  What's clearer?

Reply:
I think removing it would make it more clear.